### ISSUE TITLE
Audit link/button labels: ensure a11y for reused labels (ie, "View more")

### ISSUE THREAD

**Issue by nucleogenesis** [learningequality/kolibri#10620]
## Overview

@marcellamaki presented a11y notes regarding link usage during a KDS Tactical.

One point raised by @radinamatic was that some labeling can be reused (ie, things like "View more") in a way that provides context visually that can be missing for those using screen readers. 

It was suggested, perhaps, that we can provide better contextual help for users that use screen readers by including contextual information within that link tag's metadata. For example, we may include the title of a section (ie, a topic name "Geometry") which has a "View more" button below it.

## Deliverables

- [ ] Produce a list of places within Kolibri where labels of this kind are reused in different contexts.
- [ ] Include annotations about what pieces of surrounding context sighted users have available that screen reader users would not have.
- [ ] Close this issue with the creation of a new issue detailing the steps necessary to remedy the found examples
